Sources/Swiftly/ListAvailable.swift

```diff
@@ -51,11 +51,13 @@ struct ListAvailable: SwiftlyCommand {
 
         switch selector {
         case let .snapshot(branch, _):
-            if case let .release(major, _) = branch, major < 6 {
-                throw Error(message: "Listing available snapshots previous to 6.0 is not supported.")
+            do {
+                tc = try await SwiftlyCore.httpClient.getSnapshotToolchains(platform: config.platform, branch: branch).map { ToolchainVersion.snapshot($0) }
+            } catch let branchNotFoundError as SwiftlyHTTPClient.SnapshotBranchNotFoundError {
+                throw Error(message: "The snapshot branch \(branchNotFoundError.branch) was not found on swift.org. Note that snapshot toolchains are only available for the current `main` release and the previous x.y (major.minor) release.")
```

Comment:
Might want to use the word `latest` here as opposed to `previous`. In the update command we say

>""The branch \(branchNotFoundErr.branch) doesn't have any snapshots available on swift.org so this snapshot build cannot be updated. It is possible that there has been a new release on swift.org and the previous release snapshot are no longer available. Install a fresh snapshot toolchain from the either the latest release x.y (major.minor) or from the main branch.""

Where we tell users "previous" are gone, install "latest". But here we say that main and "previous" are available.

Basically just some very small confusion around previous != latest